test/test_inverse.jl

```diff
@@ -56,9 +56,12 @@ InverseFunctions.inverse(f::Bar) = Bar(inv(f.A))
     @test_throws DomainError inverse(Base.Fix1(*, 0))
     @test_throws DomainError inverse(Base.Fix2(^, 0))
     @test_throws DomainError inverse(Base.Fix1(log, 2))(-5)
-    InverseFunctions.test_inverse(Base.Fix1(log, 2), -5 + 0im)
+    @test inverse(Base.Fix1(log, 2))(-5 + 0im) == 0.03125 + 0im
```

Comment:
Same here, was the test incorrect?

Reply:
Oh, overlooked that one during review - I agree, `InverseFunctions.test_inverse` should be kept if at all possible (or fixed, if there's a problem with it).

Reply:
If you want to test `inverse(log)`, couldn't you use `test_inverse(inverse(log), complex(-5))`?